Microsoft.Identity.Web/WebApiStartupHelpers.cs

```diff
@@ -123,30 +98,12 @@ public static IServiceCollection AddProtectedApiCallsWebApis(this IServiceCollec
                 // flow will be delayed (lazy construction of MSAL's application
                 options.Events.OnTokenValidated = async context =>
                 {
-                    if (scopes != null && scopes.Any())
-                    {
-                        var tokenAcquisition = context.HttpContext.RequestServices.GetRequiredService<ITokenAcquisition>();
-                        context.Success();
-                        tokenAcquisition.AddAccountToCacheFromJwt(context, scopes);
-                    }
-                    else
-                    {
-                        context.Success();
-
-                        // Todo : rather use options.SaveToken?
-                        JwtSecurityToken jwtSecurityToken = context.SecurityToken as JwtSecurityToken;
-                        if (jwtSecurityToken != null)
-                        {
-                            string rawData = (jwtSecurityToken.InnerToken != null) ? jwtSecurityToken.InnerToken.RawData : jwtSecurityToken.RawData;
-                            (context.Principal.Identity as ClaimsIdentity).AddClaim(new Claim("jwt", rawData));
-                        }
-                    }
-                    // Adds the token to the cache, and also handles the incremental consent and claim challenges
+                    context.Success();
```

Comment:
If we just return success, we don't need to subscribe to the OnTokenValidated event, I'd think.